app/src/pages/admin/projectEventsPage/eventsGrid/eventsGrid.jsx

```diff
@@ -0,0 +1,184 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { injectIntl, intlShape, defineMessages } from 'react-intl';
+import { Grid } from 'components/main/grid';
+import { AbsRelTime } from 'components/main/absRelTime';
+import styles from './eventsGrid.scss';
+
+const cx = classNames.bind(styles);
+const messages = defineMessages({
+  timeCol: { id: 'EventsGrid.timeCol', defaultMessage: 'Time' },
+  userCol: { id: 'EventsGrid.userCol', defaultMessage: 'User' },
+  actionCol: { id: 'EventsGrid.actionCol', defaultMessage: 'Action' },
+  objectTypeCol: { id: 'EventsGrid.objectTypeCol', defaultMessage: 'Object Type' },
+  objectNameCol: { id: 'EventsGrid.objectNameCol', defaultMessage: 'Object Name' },
+  oldValueCol: { id: 'EventsGrid.oldValueCol', defaultMessage: 'Old Value' },
+  newValueCol: { id: 'EventsGrid.newValueCol', defaultMessage: 'New Value' },
+});
+
+const TimeColumn = ({ className, value }) => (
+  <div className={cx('time-col', className)}>
+    <AbsRelTime startTime={value.lastModifiedDate} />
+  </div>
+);
+TimeColumn.propTypes = {
+  className: PropTypes.string.isRequired,
+  value: PropTypes.object,
+};
+TimeColumn.defaultProps = {
+  value: {},
+};
+
+const UserColumn = ({ className, value }) => (
+  <div className={cx('user-col', className)}>{value.userRef}</div>
+);
+UserColumn.propTypes = {
+  className: PropTypes.string.isRequired,
+  value: PropTypes.object,
+};
+UserColumn.defaultProps = {
+  value: {},
+};
+
+const ActionColumn = ({ className, value }) => {
+  const formattedActionType = value.actionType.split(/(?=[A-Z])/).join(' ');
```

Comment:
I think we need to match each type of action (`actionType`) with the corresponding message in different languages (see screenshot below for version 4.3 of RP).
![image](https://user-images.githubusercontent.com/31861805/51766413-8dcdf800-20eb-11e9-9898-bebf09e54b3e.png)
I think we need a special activity messages map for this case.
You can find some necessary messages at the `src/components/widgets/tables/projectActivity` & `..../projectActivity/activities`.
@DiscoElevator what do you think?

Reply:
Sounds good!